docs/VictoriaLogs/backup.md

```diff
@@ -0,0 +1,62 @@
+---
+weight: 6
+title: Backup and Restore
+menu:
+  docs:
+    identifier: "victorialogs-backup-and-restore"
+    parent: "victorialogs"
+    weight: 6
+    title: Backup
+aliases:
+- /VictoriaLogs/backup-and-restore.html
+---
+
+## Backup
+VictoriaLogs currently does not have a snapshot feature and a tool like vmbackup as VictoriaMetrics does. 
+So backing up VictoriaLogs requires manually executing the `rsync` command. 
+
+The files in VictoriaLogs have the following properties:
+- All the data files are immutable. Small metadata files can be modified.
+- Old data files are periodically merged into new data files.
+
+Therefore, for a complete data backup, you need to run the `rsync` command **twice**.
+
+```bash
```

Comment:
this has artifacts on render for some reason. Could you please verify it is renderred correctly when making ` make docs-debug`?